codecov.yaml

```diff
@@ -0,0 +1,24 @@
+codecov:
+  token: a2b1c854-8627-4747-89cb-c312da33227d
+  require_ci_to_pass: no
+
+coverage:
+  precision: 2
+  round: down
+  range: "0...100"
```

Comment:
is the plan to change this later?

Reply:
My bad, we should also use the default value here, 70...100.